x/concentrated-liquidity/types/keyspace.md

```diff
@@ -0,0 +1,107 @@
+# CL keyspace
+
+This document defines the key space format used in CL, and guarantees that are expected to hold.
+It was created to more easily reason about key space bugs that we were facing.
+
+If || is seen outside of a `backtick` quotation, it means raw appending.
```

Comment:
This || is different from the accum key separator? Can we have a different notation to avoid confusion?

Reply:
Its whether its in backtick or not. || is universal string append notation